test/extended/util/annotate/generated/zz_generated.annotations.go

```diff
@@ -719,7 +719,7 @@ var annotations = map[string]string{
 
 	"[Top Level] [sig-apps][Feature:DeploymentConfig] deploymentconfigs paused should disable actions on deployments": "should disable actions on deployments [Suite:openshift/conformance/parallel]",
 
```

Comment:
Why are all of these changing?

Reply:
Noted in https://github.com/openshift/origin/pull/25562#issuecomment-700260359, no idea — gotta figure out what's going on there.

Reply:
https://github.com/openshift/kubernetes/pull/377